CommonTools/RecoAlgos/plugins/IsoTrackSelector.cc

```diff
@@ -0,0 +1,12 @@
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "CommonTools/UtilAlgos/interface/SingleObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
+#include "DataFormats/PatCandidates/interface/IsolatedTrack.h"
+
+
+ typedef SingleObjectSelector<
+           pat::IsolatedTrackCollection, 
+           StringCutObjectSelector<pat::IsolatedTrack> 
+         > IsoTrackSelector;
```

Comment:
PhysicsTools/PatAlgos/plugins/PATObjectSelector.h seems like a more appropriate place to define this.